### PR TITLE
fix: content types sync in dev

### PR DESCRIPTION
**Pull request by florian-lefebvre**
## Changes

- Try fix a path issue

## Testing

Should pass

## Docs

N/A

### CODE REVIEW

**Comment by florian-lefebvre** [comment]
!preview content-types-dev

**Comment by florian-lefebvre** [comment]
Also approved by user

File: packages/astro/src/core/sync/index.ts [hunk 1]
```diff
@@ -124,7 +125,9 @@ export async function syncInternal({
 			});
 			await contentLayer.sync();
 			settings.timer.end('Sync content layer');
-		} else if (fs.existsSync(getContentPaths(settings.config, fs).contentDir.href)) {
+		} else if (
+			fs.existsSync(normalizePath(fileURLToPath(getContentPaths(settings.config, fs).contentDir)))
```
Comment (bluwy) [inline packages/astro/src/core/sync/index.ts:129]: We shouldn't need `normalizePath` here. That's not what it's for. Shouldn't the `fileUrlToPath` be enough?

**Comment by florian-lefebvre** [comment]
!preview content-types-dev

**Pull request merged**